internal/cmd/schema-registry/command_cluster_update.go

```diff
@@ -73,7 +77,55 @@ func (c *command) updateTopLevelCompatibility(cmd *cobra.Command) error {
 		return err
 	}
 
-	updateReq := srsdk.ConfigUpdateRequest{Compatibility: strings.ToUpper(compatibility)}
+	compatibilityGroup, err := cmd.Flags().GetString("compatibility-group")
+	if err != nil {
+		return err
+	}
+
+	metadataPath, err := cmd.Flags().GetString("metadata-defaults")
```

Comment:
We try to keep a 1-1 matching between flag names and variable names
```suggestion
	metadataDefaults, err := cmd.Flags().GetString("metadata-defaults")
```